src/main/java/com/google/devtools/build/lib/analysis/config/CoreOptions.java

```diff
@@ -624,6 +624,19 @@ public OutputDirectoryNamingSchemeConverter() {
               + "Bazel release.")
   public List<String> defaultFeatures;
 
+  @Option(
+      name = "host_features",
+      allowMultiple = true,
+      defaultValue = "null",
+      documentationCategory = OptionDocumentationCategory.OUTPUT_PARAMETERS,
+      effectTags = {OptionEffectTag.CHANGES_INPUTS, OptionEffectTag.AFFECTS_OUTPUTS},
+      help =
+          "The given features will be enabled or disabled by default for packages "
+              + "built in the host or exec configurations. "
+              + "Specifying -<feature> will disable the feature globally. "
```

Comment:
"globally" can be confusing here given that the previous sentence take about the "exec configuration" only.